Dockerfile

```diff
@@ -10,11 +10,22 @@ RUN set -ex && \
     rm -f /usr/libexec/gcc/x86_64-alpine-linux-musl/6.4.0/lto-wrapper && \
     rm -f /usr/bin/x86_64-alpine-linux-musl-gcj
 
+RUN set -ex && \
+    apk add --no-cache gcc g++ musl-dev python3 openjdk17 iptables ip6tables && \
+    apk add --no-cache --virtual .build-deps bash curl git && \
```

Comment:
Why do you want to add `bash`, `curl` and `git`?

Reply:
sorry, only curl will be needed to download the tar for latest go compiler.